test/Feature/TargetMismatch.c

```diff
@@ -0,0 +1,8 @@
+// REQUIRES: not-darwin
+// RUN: %clang %s -m32 -emit-llvm %O0opt -c -o %t1.bc
+// RUN: rm -rf %t.klee-out
+// RUN: %klee --output-dir=%t.klee-out --exit-on-error %t1.bc 2>&1 | FileCheck %s
```

Comment:
```suggestion
// RUN: %klee --output-dir=%t.klee-out %t1.bc 2>%t.log || /bin/true
// RUN: FileCheck %s --input-file %t.log
```
KLEE segfaults - but we expected this as the host triple and test triple are different.
Use `|| /bin/true` to make sure this line is successful anyway and just `FileCheck` for the warning message.